_pages/es/index.md

```diff
@@ -15,7 +15,7 @@ A campaign for Mozillians around the world to have impact in areas key to Mozill
 
 {: #area-deveng .area}
 [![image](/asserts/img/development.png)
-Empower developer communities]({{ "developer-engagement" | prepend: page.url | prepend: site.baseurl }})
+Empower developer communities](/es/developer-engagement/)
```

Comment:
I tried (like you could see in other files), but didn't work on this one (maybe I didn't wait enough, other changes take a lot to be reflected in the website)